README.md

```diff
@@ -54,17 +54,19 @@ Start with a simple example:
 
 ```
 $ cat << EOF > test.py
-a = 1
-b = 2
-if a == a:  # oops, supposed to be a == b
-    print('sgrep test')
+def get_node(node_id, nodes):
```

Comment:
This example felt less contrived while still staying relatively small and self-contained.